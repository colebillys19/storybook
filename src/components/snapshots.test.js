import initStoryshots from '@storybook/addon-storyshots';

// automatically creates a snapshot for each of the stories
// requires dev dependencies @storybook/addon-storyshots react-test-renderer require-context.macro
// react-test-renderer version should be the same as react and react-dom packages
initStoryshots();
