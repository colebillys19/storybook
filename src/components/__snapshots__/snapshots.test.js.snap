// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Banners|AlertBanner default 1`] = `
<section
  className="sc-bxivhb eMtVEh"
>
  <div
    className="sc-EHOje jtyqUw"
  >
    <p>
      alert text
      <button
        className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-raised-15 sc-ifAKCX bqdATS sc-bdVaJa iRGXdW"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-2 label"
        >
          button text
        </span>
      </button>
    </p>
  </div>
</section>
`;

exports[`Storyshots Banners|ErrorSuccessBanner error 1`] = `
<div
  className="sc-gzVnrw avLPF"
  color="#C1363F"
>
  <div
    className="sc-htoDjs kYYNpa"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-30 root styled-icon sc-VigVT bkqdLp"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
      />
    </svg>
  </div>
  <div
    className="sc-iwsKbI dwDcyq"
  >
    error
  </div>
  <button
    className="MuiButtonBase-root-65 MuiButton-root-39 MuiButton-contained-50 MuiButton-raised-53 sc-dnqmqq ibEVtO sc-bdVaJa iRGXdW"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-40 label"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-30 root styled-icon sc-gZMcBi fwoZLL"
        focusable="false"
        role="presentation"
        viewBox="0 0 500 500"
      >
        <path
          clipRule="evenodd"
          d="M483 456.3L275.7 249 483 41.7 456.3 15 249 222.3 41.7 15 15 41.7 222.3 249 15 456.3 41.7 483 249 275.7 456.3 483z"
          fillRule="evenodd"
        />
      </svg>
      
    </span>
  </button>
</div>
`;

exports[`Storyshots Banners|ErrorSuccessBanner success 1`] = `
<div
  className="sc-gzVnrw dHKoqK"
  color="#5DBA9C"
>
  <div
    className="sc-htoDjs kYYNpa"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-68 root styled-icon sc-gqjmRU kVDmhf"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0V0zm0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M16.59 7.58L10 14.17l-3.59-3.58L5 12l5 5 8-8zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
      />
    </svg>
  </div>
  <div
    className="sc-iwsKbI dwDcyq"
  >
    success
  </div>
  <button
    className="MuiButtonBase-root-103 MuiButton-root-77 MuiButton-contained-88 MuiButton-raised-91 sc-dnqmqq ibEVtO sc-bdVaJa iRGXdW"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-78 label"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-68 root styled-icon sc-gZMcBi fwoZLL"
        focusable="false"
        role="presentation"
        viewBox="0 0 500 500"
      >
        <path
          clipRule="evenodd"
          d="M483 456.3L275.7 249 483 41.7 456.3 15 249 222.3 41.7 15 15 41.7 222.3 249 15 456.3 41.7 483 249 275.7 456.3 483z"
          fillRule="evenodd"
        />
      </svg>
      
    </span>
  </button>
</div>
`;

exports[`Storyshots Buttons|BaseButton default 1`] = `
<button
  className="MuiButtonBase-root-132 MuiButton-root-106 MuiButton-contained-117 MuiButton-raised-120 sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-107 label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-135 root styled-icon sc-kkGfuU cGDOnE"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
    label text
  </span>
</button>
`;

exports[`Storyshots Buttons|FlatIconButton default 1`] = `
<button
  aria-describedby={null}
  className="MuiButtonBase-root-178 MuiButton-root-152 MuiButton-contained-163 MuiButton-raised-166 sc-bZQynM jlZJzO sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  title="tooltip text"
  type="button"
>
  <span
    className="MuiButton-label-153 label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-181 root styled-icon sc-iAyFgw kGAJlA"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
    label text
  </span>
</button>
`;

exports[`Storyshots Buttons|GhostButton default 1`] = `
<button
  className="MuiButtonBase-root-216 MuiButton-root-190 MuiButton-contained-201 MuiButton-raised-204 sc-jzJRlG hTvvkz sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-191 label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-219 root styled-icon sc-hSdWYo kuhIRg"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
    label text
  </span>
</button>
`;

exports[`Storyshots Buttons|IconButton default 1`] = `
<button
  className="MuiButtonBase-root-234 MuiIconButton-root-228"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiIconButton-label-233"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-237 root styled-icon sc-eHgmQL gcRwue"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
  </span>
  <span
    className="MuiTouchRipple-root-246"
  />
</button>
`;

exports[`Storyshots Buttons|IconButton disabled 1`] = `
<button
  className="MuiButtonBase-root-259 MuiButtonBase-disabled-260 MuiIconButton-root-253 MuiIconButton-disabled-257"
  disabled={true}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="-1"
  type="button"
>
  <span
    className="MuiIconButton-label-258"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-262 root styled-icon sc-cvbbAY lfKHPD"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
  </span>
</button>
`;

exports[`Storyshots Buttons|PrimaryButton default 1`] = `
<button
  className="MuiButtonBase-root-297 MuiButton-root-271 MuiButton-contained-282 MuiButton-raised-285 sc-jTzLTM lcwPIJ sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-272 label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-300 root styled-icon sc-jWBwVP bGQksY"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
    label text
  </span>
</button>
`;

exports[`Storyshots Buttons|SecondaryButton default 1`] = `
<button
  className="MuiButtonBase-root-335 MuiButton-root-309 MuiButton-contained-320 MuiButton-raised-323 sc-fjdhpX dQkZpA sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-310 label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-338 root styled-icon sc-brqgnP hzGMET"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
    label text
  </span>
</button>
`;

exports[`Storyshots Buttons|TextButton default 1`] = `
<button
  className="MuiButtonBase-root-373 MuiButton-root-347 MuiButton-contained-358 MuiButton-raised-361 sc-htpNat fzSZoe sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-348 label"
  >
    label text
  </span>
</button>
`;

exports[`Storyshots Buttons|TooltipButton default 1`] = `
<button
  aria-describedby={null}
  className="MuiButtonBase-root-410 MuiButton-root-384 MuiButton-contained-395 MuiButton-raised-398 sc-bdVaJa iRGXdW"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  title="tooltip text"
  type="button"
>
  <span
    className="MuiButton-label-385 label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-413 root styled-icon sc-cMljjf eIjVek"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3H4.99c-1.11 0-1.98.89-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.11-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10z"
      />
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
    </svg>
    label text
  </span>
</button>
`;

exports[`Storyshots Cards|BaseCard default 1`] = `
<div
  className="MuiPaper-root-423 MuiPaper-elevation1-426 MuiPaper-rounded-424 MuiCard-root-422 card sc-cSHVUG dvYpwr"
  color="#310078"
>
  <div
    style={
      Object {
        "padding": "1rem",
      }
    }
  >
    card text
  </div>
</div>
`;

exports[`Storyshots Cards|BaseCard image 1`] = `
<div
  className="MuiPaper-root-479 MuiPaper-elevation1-482 MuiPaper-rounded-480 MuiCard-root-478 card sc-cSHVUG dvYpwr"
  color="#310078"
>
  <div
    style={
      Object {
        "padding": "1rem",
      }
    }
  >
    <img
      alt="placeholder"
      src="imagePlaceholder.svg"
      style={
        Object {
          "width": "10rem",
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots Cards|BaseCard title 1`] = `
<div
  className="MuiPaper-root-451 MuiPaper-elevation1-454 MuiPaper-rounded-452 MuiCard-root-450 card sc-cSHVUG dvYpwr"
  color="#310078"
>
  <div
    style={
      Object {
        "padding": "1rem",
      }
    }
  >
    <h1
      className="sc-kAzzGY dFlnQs"
    >
      card text
    </h1>
  </div>
</div>
`;

exports[`Storyshots Headings|H1 default 1`] = `
<h1
  className="sc-chPdSV iujesU"
  style={
    Object {
      "color": "#464A53",
    }
  }
>
  H1 header text
</h1>
`;

exports[`Storyshots Headings|H2 default 1`] = `
<h2
  className="sc-kgoBCf kFsPKU"
  style={
    Object {
      "color": "#464A53",
    }
  }
>
  H2 header text
</h2>
`;

exports[`Storyshots Headings|H3 default 1`] = `
<h3
  className="sc-kGXeez dyhKDK"
  style={
    Object {
      "color": "#464A53",
    }
  }
>
  H3 header text
</h3>
`;

exports[`Storyshots Links|BaseLink default 1`] = `
<a
  className="sc-kpOJdX emQGug"
  classes={
    Object {
      "root": "link",
    }
  }
  href="/path"
  onClick={[Function]}
>
  Base Link
</a>
`;

exports[`Storyshots Links|BaseLink icon 1`] = `
<a
  className="sc-kpOJdX emQGug"
  classes={
    Object {
      "root": "link",
    }
  }
  href="/path"
  onClick={[Function]}
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root-506 root styled-icon sc-jAaTju fcuGzG"
    focusable="false"
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
    <path
      d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
    />
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
  </svg>
  Base Link
</a>
`;

exports[`Storyshots Links|GhostButtonLink default 1`] = `
<a
  className="sc-dxgOiQ cQUnhg sc-kpOJdX emQGug"
  classes={
    Object {
      "root": "link",
    }
  }
  href="/path"
  onClick={[Function]}
>
  Ghost Button Link
</a>
`;

exports[`Storyshots Links|GhostButtonLink icon 1`] = `
<a
  className="sc-dxgOiQ cQUnhg sc-kpOJdX emQGug"
  classes={
    Object {
      "root": "link",
    }
  }
  href="/path"
  onClick={[Function]}
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root-515 root styled-icon sc-jDwBTQ gmWAQx"
    focusable="false"
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
    <path
      d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
    />
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
  </svg>
  Ghost Button Link
</a>
`;

exports[`Storyshots List Items|ListItem button 1`] = `
<div
  className="MuiButtonBase-root-590 MuiListItem-root-578 MuiListItem-default-581 MuiListItem-gutters-586 MuiListItem-button-587"
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  role="button"
  tabIndex="0"
>
  <div
    className="MuiListItemText-root-593 item-text"
  >
    <span
      className="MuiTypography-root-599 MuiTypography-body1-608 MuiListItemText-primary-596"
    >
      List Item Button
    </span>
  </div>
  <span
    className="MuiTouchRipple-root-635"
  />
</div>
`;

exports[`Storyshots List Items|ListItem default 1`] = `
<li
  className="MuiListItem-root-524 MuiListItem-default-527 MuiListItem-gutters-532"
  disabled={false}
>
  <div
    className="MuiListItemText-root-536 item-text"
  >
    <span
      className="MuiTypography-root-542 MuiTypography-body1-551 MuiListItemText-primary-539"
    >
      Default List Item
    </span>
  </div>
</li>
`;

exports[`Storyshots List Items|ListItem disabled 1`] = `
<li
  className="MuiListItem-root-642 MuiListItem-default-645 MuiListItem-gutters-650 MuiListItem-disabled-648"
  disabled={true}
  onClick={[Function]}
>
  <div
    className="MuiListItemText-root-654 item-text"
  >
    <span
      className="MuiTypography-root-660 MuiTypography-body1-669 MuiListItemText-primary-657"
    >
      List Item Disabled
    </span>
  </div>
</li>
`;

exports[`Storyshots List Items|ListItem icon 1`] = `
<li
  className="MuiListItem-root-696 MuiListItem-default-699 MuiListItem-gutters-704"
  disabled={false}
  onClick={[Function]}
>
  <div
    className="MuiListItemText-root-708 item-text"
  >
    <span
      className="MuiTypography-root-714 MuiTypography-body1-723 MuiListItemText-primary-711"
    >
      List Item w/ Icon
    </span>
  </div>
  <div
    className="MuiListItemIcon-root-750"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-751 root styled-icon sc-gPEVay fqnzzU"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
      />
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
    </svg>
  </div>
</li>
`;

exports[`Storyshots List Items|ListItem selected 1`] = `
<li
  className="MuiListItem-root-760 MuiListItem-default-763 MuiListItem-gutters-768 MuiListItem-selected-771"
  disabled={false}
  onClick={[Function]}
>
  <div
    className="MuiListItemText-root-772 item-text"
  >
    <span
      className="MuiTypography-root-778 MuiTypography-body1-787 MuiListItemText-primary-775"
    >
      List Item Selected
    </span>
  </div>
</li>
`;

exports[`Storyshots Loading Indicators|LoadingIndicator default 1`] = `
<div
  className="sc-ckVGcZ dfagEm"
>
  <section
    className="sc-hMqMXs ksMxRb"
  >
    <div
      className="sc-iRbamj cScRwx"
    />
    <div
      className="sc-jlyJG iQlOWZ"
    />
    <div
      className="sc-gipzik PSofd"
    />
    <div
      className="sc-csuQGl lnAIYV"
    />
    <div
      className="sc-Rmtcm vNIZD"
    />
    <div
      className="sc-bRBYWo cLyBkh"
    />
    <div
      className="sc-hzDkRC ficHPt"
    />
    <div
      className="sc-jhAzac dGiajx"
    />
    <div
      className="sc-fBuWsC eqBTVZ"
    />
    <div
      className="sc-fMiknA fAtOnz"
    />
    <div
      className="sc-dVhcbM bZVcIg"
    />
    <div
      className="sc-eqIVtm hgQoQi"
    />
  </section>
</div>
`;

exports[`Storyshots Loading Indicators|SmallLoadingIndicator default 1`] = `
<span
  className="sc-jKJlTe dJwQLv"
>
  <span
    className="sc-eNQAEJ eHjyYU"
  >
    <div
      className="sc-fAjcbJ gFsbFl"
    />
    <div
      className="sc-caSCKo gVaYjT"
    />
    <div
      className="sc-gisBJw geSlCx"
    />
    <div
      className="sc-kjoXOD iosaVH"
    />
    <div
      className="sc-cHGsZl hJpKhB"
    />
    <div
      className="sc-TOsTZ ifFKKG"
    />
    <div
      className="sc-kgAjT cnpqRY"
    />
    <div
      className="sc-cJSrbW zlgOq"
    />
    <div
      className="sc-ksYbfQ PwgZO"
    />
    <div
      className="sc-hmzhuo cuwoob"
    />
    <div
      className="sc-frDJqD beGBbd"
    />
    <div
      className="sc-kvZOFW dgabsT"
    />
  </span>
</span>
`;

exports[`Storyshots Paper|BasePaper default 1`] = `
<div
  className="MuiPaper-root-814 MuiPaper-elevation2-818 MuiPaper-rounded-815"
>
  <div
    style={
      Object {
        "padding": "20px",
      }
    }
  >
    paper content
  </div>
</div>
`;

exports[`Storyshots Paper|InsetPaper default 1`] = `
<div
  className="MuiPaper-root-841 MuiPaper-elevation2-845 MuiPaper-rounded-842 sc-kEYyzF cIlGpN"
>
  <div
    style={
      Object {
        "padding": "20px",
      }
    }
  >
    paper content
  </div>
</div>
`;
